src/main/java/org/folio/service/manager/export/ExportResult.java

```diff
@@ -1,5 +1,86 @@
 package org.folio.service.manager.export;
 
-public enum ExportResult {
-  COMPLETED, IN_PROGRESS, ERROR;
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
+import org.folio.util.ErrorCode;
+
+@DataObject(generateConverter = true)
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class ExportResult {
```

Comment:
There is a discussion held about how to pass an error code from ExportManager to InputDataManager in order to let the InputDataManager update job with error cause.
_@DataObject_ annotation is used to make ExportResult be serializable on an event bus.